jsk_perception/launch/cmt_sample.launch

```diff
@@ -0,0 +1,60 @@
+<launch>
```

Comment:
Should be renamed as `sample/sample_concensus_tracking.launch`?
and it can be written without pointcloud, and it should be, isn't it?